frontend/src/services/eventHandlerService.js

```diff
@@ -1,18 +1,19 @@
-import { GlobalStore } from "@/modules/common/store/globalstore/index";
 import { EventBus } from "../eventBus.js";
+import EventHandlerFactory from "./eventHandlers/eventHandlerFactory";
 
+// Service for handling websocket events
 export default class EventHandlerService {
     handle(api, payload) {
-        if (api === "hub") {
-            this.handleHubEvent(payload);
+        console.log("----- Event Received -----");
+        console.log(api);
+        console.log(payload);
+
+        let eventHandlerFactory = new EventHandlerFactory();
```

Comment:
I don't think we need to be creating a new instance every time. Maybe the EventHandlerService should maintain a map of handlers

Reply:
Changed it so it is only created once